### PR TITLE
feat : wishlist page

### PR DESCRIPTION
**Pull request by Kavita539**
Added wishlist page

### CODE REVIEW

**Review by way2nizam — APPROVED** [review]
Looks good.

**Review by sharathnair9999 — APPROVED** [review]

File: pages/wishlist-page/wishlist-page.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@import url(/stylesheets/header.css);
```
Comment (ArjunGTX) [inline pages/wishlist-page/wishlist-page.css]: Minimize the usage of `@imports`.  An extra HTTP request is created of each css `@import`. Use `link` tag instead.

**Review by ArjunGTX — APPROVED** [review]
Everything else looks good 👍

File: pages/wishlist-page/wishlist-page.css (hunk 1 above)
Comment (Kavita539) [reply pages/wishlist-page/wishlist-page.css]: ok will keep that in mind!

**Pull request merged**